### PR TITLE
Initialize Config Windows Network for Windows Namespace

### PR DESCRIPTION
**Pull request by madhanrm**
Initialize the windows network config during Network Namespace initialization

Signed-off-by: Madhan Raj Mookkandy <madhanm@microsoft.com>

### CODE REVIEW

**Comment by madhanrm** [comment]
@vbatts Can you provide some clue on how to fix the failing test?

**Comment by jterry75** [comment]
@crosbymichael - Can you take a look at #667 and this one. Would be helpful for us to get these in to finish the CRI work.

**Comment by madhanrm** [comment]
@q384566678  can you take a look?

File: generate/generate.go [hunk 1]
```diff
@@ -1557,6 +1557,12 @@ func (g *Generator) SetWindowsNetwork(network rspec.WindowsNetwork) {
 	g.Config.Windows.Network = &network
 }
 
+// SetWindowsNetworkNamespace sets g.Config.Windows.Network.NetworkNamespace
+func (g *Generator) SetWindowsNetworkNamespace(path string) {
+	g.initConfigWindowsNetwork()
+	g.Config.Windows.Network.NetworkNamespace = path
+}
+
```
Comment (q384566678) [inline generate/generate.go]: You need to add the corresponding option to the `generate` command to call the `SetWindowsNetworkNamespace` function.

**Comment by madhanrm** [comment]
@q384566678  Can you take a look?

**Comment by q384566678** [comment]
Please update the relevant instructions to `man/oci-runtime-tool-generate.1.md` and `completions/bash/oci-runtime-tool`.

**Comment by q384566678** [comment]
LGTM

**Pull request merged**